pom.xml

```diff
@@ -359,6 +359,7 @@
             <exclude>**/*.pcap</exclude>
             <exclude>**/*.log1</exclude>
             <exclude>**/*.log2</exclude>
+            <exclude>**/*.sav</exclude>
```

Comment:
Please also update the exclusion list for `license-maven-plugin`.

Reply:
Done.